README.md

```diff
@@ -25,6 +25,8 @@ npx ts-node examples/opensea.ts
 # You will be prompted to provide a collectionSlug
 ```
 
+A collection slug identifies the collection at OpenSea. To get it you have to go to [testnet.opensea](https://testnets.opensea.io/). An example slug you can use is: [the-monkey-chainz](https://testnets.opensea.io/collection/the-monkey-chainz).
```

Comment:
Hmm, you're right about that. However, I'm a bit unclear on what exactly I need to do. Could you please provide more details?

Reply:
So, precisely I think it would be nice to have:

1. A new file `examples/README.md`
   This file would contain some elaboration about the examples contained within the directory. 
   You could start it off by cutting out the "Other Examples" section of the current readme: https://github.com/Mintbase/near-ca/blob/a1d3c557e19b68e13587a0d520b82813179fa1e3/README.md?plain=1#L73-L87 
2. Add in the content you've included about the collection slug as well.
3. Replace the Old Usage Section with a reference to the `examples/README.md`
    Something like 
    > see `[here](examples/README.md)` for some example scripting with this library.

Reply:
I attempted to separate the READMEs for better organization. However, there might be some mistakes, so please review the changes and let me know if everything is correct.

Reply:
LGTM, hvala!